broccoli-shop/src/test/java/org/mytest/DummyTest.java

```diff
@@ -0,0 +1,26 @@
+package org.mytest;
+
+import org.junit.Test;
+
+public class DummyTest {
+
+    @Test
+    public void test() {
+        int a = 1;
+        a *= 10;
+        assert a == 10;
+    }
+
+    @Test
+    public void duplicatedTest() {
+        int a = 1;
+        a *= 10;
+        assert a == 10;
+    }
+
+    @Test
+    public void stupidTest() {
+        assert 1 == 1;
+        assert true == true;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Correct one of the identical sub-expressions on both sides of operator "==" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1764)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "true" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1125)